game/src/game/game_player/timing_judge.rs

```diff
@@ -158,8 +166,13 @@ impl TimingJudge {
 
             // if it's processable note, calculate accuracy
             if let Some(hit_timing) = i.hit_timing {
-                let note_accuracy =
-                    note_accuracy_from_time_difference(hit_timing as i64 - precise_timing as i64);
+                let difference_abs = (hit_timing as i64 - precise_timing as i64).abs();
+
+                // calculte score by the accuracy
+                self.score += ((f64::abs(160.0 - difference_abs.clamp(10, 160) as f64) / 150.0)
```

Comment:
Just one evil idea: How about applying ease-in function to `((f64::abs(160.0 - difference_abs.clamp(10, 160) as f64) / 150.0)`?

(Ignore this if you dislike it :D)